build.gradle

```diff
@@ -16,7 +16,7 @@ defaultTasks 'clean', 'licenseFormat', 'build'
 apply plugin: 'java'
 
 group = 'org.spongepowered'
-version = '0.8.2-SNAPSHOT'
+version = '0.8.3-SNAPSHOT'
```

Comment:
Why? 0.8.2 wasn't released yet, so not sure why we'd skip it. (Although the next release will be probably 0.9.0, due to the new ore deploy plugin)

There will be a separate commit to bump the version, though, so please don't do this as part of this PR.